ckanext/canada/controller.py

```diff
@@ -229,6 +230,9 @@ def datatable(self, resource_name, resource_id):
                         )
                     )
                 )
+        # Add empty column for checkboxes #
+        for row in aadata:
+            row.insert(0,u'')
```

Comment:
do this in the creation of the original aadata list to save a bunch of unnecessary data copying